kernel/packages/unity-interface/dcl.ts

```diff
@@ -116,6 +116,35 @@ const positionEvent = {
   mousePosition: Vector3.Zero()
 }
 
+/////////////////////////////////// AUDIO STREAMING ///////////////////////////////////
+
+const audioStreamSource = new Audio()
+
+teleportObservable.add(() => {
+  audioStreamSource.pause()
+})
+
+async function setAudioStream(url: string, play: boolean, volume: number) {
```

Comment:
This code shouldn't be more maintainable if we move this method to another file? 
`dcl.ts` should try to contain the basic messaging plumbing only.

Reply:
This is all we need for now. I will totally agree to move it to other file in the future if we need to add more features or extend it functionality